lib/services/configuration.js

```diff
@@ -43,6 +43,9 @@ function configurationServiceFactory(
         } else if (fs.existsSync(Constants.Configuration.Files.OnRack)) {
             //TODO: remove once OnRack build process adjusts for changes.
             nconf.file('global', Constants.Configuration.Files.OnRack);
+        } else {
+          console.error('Failed to load configuration file:',
+            Constants.Configuration.Files.Global);
```

Comment:
I don't think so. Possibly, I think the logger need certain configuration properties before it will work. I tried using it here when I was debugging and it was failing.

I added that when I was trying to mount `/opt/monorail/config.json` for RackHD docker containers. Took me a while to realize the config was missing. This made my life much easier.